deployment/IMPORT_INSTRUCTIONS.md

```diff
@@ -0,0 +1,37 @@
+# Import Instructions
+
+## From `pete-sls` branch:
+
+```sh
+rm -rf node_modules
+yarn install
+./run update-env
+# COMMENT OUT LOGGING_BUCKET, TEMPLATE_BUCKET, WP_FORM_BUCKET, SAR_FORM_BUCKET and values for short-circuiting SSM in .env file
+./run deploy --stage <YOUR_BRANCH_NAME>
+
+# cloudfront.Distribution -
+# cognito.UserPool -
+
+# manually dissassociate web acl in app-api-<YOUR_BRANCH_NAME>
+./run destroy --stage <YOUR_BRANCH_NAME>
+
+# manually remove bucket policy from database-<stage>-wp
+# manually remove bucket uploads-<stage>-avscan-671853096380
+# manually remove bucket policy from database-<stage>-sar
+
```

Comment:
I'm curious how the bucket importing works, do you have to note their names for import or does it just find them if your stage is named correctly (how it finds the dynamo tables for example)?

Reply:
Those manual removals I didn't have to do last time.  So that could likely be removed from the instructions.  The bucket imports are almost like the dynamodb tables, but the names don't always populate, sometimes you need to provide.  You will see I had to add more code to serverless stage destroyer for it to not empty some buckets!  :)